src/tests/ftest/util/general_utils.py

```diff
@@ -380,17 +351,29 @@ def check_task_log(message):
 
 def display_task(task):
     """Display the output for the executed task.
-
     Args:
         task (Task): a ClusterShell.Task.Task object for the executed command
+    Returns:
+        bool: if the command returned an 0 exit status on every host
```

Comment:
Missing spacing:
```suggestion
    
    Returns:
        bool: if the command returned an 0 exit status on every host
        
```

Reply:
Please advise if we need to update extra-space(line) to general_utils.py method header as well.